rest/data/aliases.go

```diff
@@ -131,6 +132,47 @@ func UpdateAliasesForSection(projectId string, updatedAliases []restModel.APIPro
 	return modified, catcher.Resolve()
 }
 
+// validateFeaturesHaveAliases returns an error if project/repo aliases are not defined for a Github/CQ feature.
+// Does not error if version control is enabled.
+func validateFeaturesHaveAliases(pRef *model.ProjectRef, aliases []restModel.APIProjectAlias) error {
+	if pRef.IsVersionControlEnabled() {
```

Comment:
can i ask for clarification on why we dont error

Reply:
>  require that projects choose either to use repo-level settings OR yaml settings (or neither, and stick with the existing format where all settings are defined in only one place)

I like this thing you mentioned in the ticket, at least as a first step until we start gathering more user feedback.